code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -39,6 +39,74 @@
 
 	return succeed_activate()
 
+// Headbite
+
+/datum/action/xeno_action/activable/headbite
+	name = "Headbite"
+	action_icon_state = "headbite"
+	mechanics_text = "Permanently kill a target."
+	use_state_flags = XACT_USE_STAGGERED|XACT_USE_FORTIFIED|XACT_USE_CRESTED //can't use while staggered, defender fortified or crest down
+	keybind_signal = COMSIG_XENOABILITY_HEADBITE
+	plasma_cost = 100
+
+/datum/action/xeno_action/activable/headbite/can_use_ability(mob/M, silent = FALSE, override_flags) //usable only on mobs, no clue what silent or override_flags does
```

Comment:
```suggestion
/datum/action/xeno_action/activable/headbite/can_use_ability(mob/M, silent = FALSE, override_flags)
```
Silent is used for updating the UI button to be greyed out, so it doesnt spam the user with messages, and override flags are in case you need to well, override